pkgs/games/crawl/default.nix

```
@@ -1,63 +1,114 @@
-{ stdenv, lib, fetchFromGitHub, fetchpatch, which, sqlite, lua5_1, perl, python3, zlib, pkg-config, ncurses
-, dejavu_fonts, libpng, SDL2, SDL2_image, SDL2_mixer, libGLU, libGL, freetype, pngcrush, advancecomp
-, tileMode ? false, enableSound ? tileMode, buildPackages
-
-# MacOS / Darwin builds
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, which
+, sqlite
+, lua5_1
+, perl
+, python3
+, zlib
+, pkg-config
+, ncurses
+, dejavu_fonts
+, libpng
+, SDL2
+, SDL2_image
+, SDL2_mixer
+, libGLU
+, libGL
+, freetype
+, pngcrush
+, advancecomp
+, tileMode ? false
+, enableSound ? tileMode
+, buildPackages
+  # MacOS / Darwin builds
 , darwin ? null
```

```suggestion
, darwin
```
That is never null even on Linux